oauth2-server/src/main/java/com/clouway/oauth2/CustomParamsParser.java

```diff
@@ -0,0 +1,33 @@
+package com.clouway.oauth2;
+
+import com.clouway.friendlyserve.Request;
+import com.clouway.friendlyserve.RequestExt;
+import com.google.common.base.Strings;
+import com.google.common.collect.Maps;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Mihail Lesikov (mlesikov@gmail.com)
+ */
+public class CustomParamsParser {
+  public Map<String, String> parse(Request request, String... exclusions) {
+
+    RequestExt req = (RequestExt) request;
+
+    List<String> exclusionsList = Arrays.asList(exclusions);
+
+    Map<String, String> params = Maps.newHashMap();
+
+    for (String key : req.params().keySet()) {
+      String value = request.param(key) == null ? "" : request.param(key);
```

Comment:
this check is unneeded request.params() will return only params that are available